mcp-server/src/core/direct-functions/add-task.js

```diff
@@ -108,150 +101,61 @@ export async function addTaskDirect(args, log, context = {}) {
 			);
 
 			// Call the addTask function with manual task data
-			const newTaskId = await addTask(
+			newTaskId = await addTask(
 				tasksPath,
-				null, // No prompt needed for manual creation
+				null, // prompt is null for manual creation
 				taskDependencies,
-				priority,
+				taskPriority,
 				{
-					mcpLog: log,
-					session
+					session,
+					mcpLog
 				},
-				'json', // Use JSON output format to prevent console output
-				null, // No custom environment
-				manualTaskData // Pass the manual task data
+				'json', // outputFormat
+				manualTaskData, // Pass the manual task data
+				false // research flag is false for manual creation
 			);
```

Comment:
meant for this addTask, instead of having a bunch of params with comma making it very difficult to identify which param position is which, would be better to make it one parameter that is an object. Easier, thats what "same here" was identifying, so it was like 
```suggestion
newTaskId = await addTask({
				tasksPath,
				null, // prompt is null for manual creation
				taskDependencies,
				taskPriority,
				{
					session,
					mcpLog
				},
				'json', // outputFormat
				manualTaskData, // Pass the manual task data
				false // research flag is false for manual creation
			});
```